src/openfluid/waresdev/WareSrcMigratorListener.hpp

```diff
@@ -50,6 +50,25 @@ namespace openfluid { namespace waresdev {
 
 class WareSrcMigratorListener : public openfluid::base::Listener
 {
+  protected:
+
+    static constexpr std::string_view DISPLAY_TEXT_PRELIMINARY_CHECKS = "Preliminary checks";
```

Comment:
This class should be agnostic of its subclasses implementations. Strings should not be stored here.